packages/portfolio-contract/src/pos-gmp.flows.ts

```diff
@@ -59,6 +68,68 @@ export const provideEVMAccount = async (
   return pk.reader.getGMPInfo(chainName) as unknown as Promise<GMPAccountInfo>; // XXX Guest/Host #9822
 };
 
+type TokenMessengerI = {
+  depositForBurn: ['uint256', 'uint32', 'bytes32', 'address'];
+};
+
+/**
+ * @see {@link https://github.com/circlefin/evm-cctp-contracts/blob/master/src/TokenMessenger.sol}
+ * 1ddc505 Dec 2022
+ */
+const TokenMessenger: TokenMessengerI = {
+  depositForBurn: ['uint256', 'uint32', 'bytes32', 'address'],
+};
```

Comment:
Is there something to be gained by this duplication of `type TokenMessengerI` and `const TokenMessenger`?

Maybe:
```suggestion
/**
 * @see {@link https://github.com/circlefin/evm-cctp-contracts/blob/master/src/TokenMessenger.sol}
 * 1ddc505 Dec 2022
 */
const TokenMessenger = {
  depositForBurn: ['uint256', 'uint32', 'bytes32', 'address'],
} as const;
```

Reply:
this leads down a `Readonly<...>` rabbit hold of uncertain depth